doc/how-to-onboard-an-acquirer.md

```diff
@@ -1,10 +1,10 @@
-# How to onboard a client
+# How to onboard an acquirer
```

Comment:
This doc is now out of date anyway because of the api endpoint @oskarwilliams378 added